app/src/main/java/io/appium/uiautomator2/server/mjpeg/MjpegScreenshotStream.java

```diff
@@ -33,8 +33,11 @@
 
 public class MjpegScreenshotStream extends Thread {
     private static final String BOUNDARY_STRING = 
-        "--BoundaryString\r\nContent-type: image/jpg\r\nContent-Length: %d\r\n\r\n";
-    private static byte[] END = "\r\n\r\n".getBytes(UTF_8);
+        "--BoundaryString\r\n" +
```

Comment:
shouldn't the boundary string be some random value?

Reply:
The string / constant portion you've referenced is not random, but once it's merged w/ the length of the `jpeg` (below). I think what we have is fine, the spec is here: https://tools.ietf.org/html/rfc7578#section-4.1